pelican_katex/rendering_katex.py

```diff
@@ -0,0 +1,322 @@
+import atexit
+import json
+import os
+import shutil
+import socket
+import struct
+import tempfile
+import time
+from contextlib import closing, contextmanager
+from pathlib import Path
+from subprocess import PIPE, Popen, TimeoutExpired
+#from .rendering import KaTeXError
```

Comment:
I wanted to import `KaTeXError` directly without a prefix, like it's done in `restructuredtext.py`, but it failed.
This would be a bit cleaner I think, but I'm not familiar enough with Python's import system.